server/dist/routes/projects.js

```diff
@@ -4,22 +4,38 @@ Object.defineProperty(exports, "__esModule", {
   value: true
 });
 
-var _dbConnection = require("../services/dbConnection.js");
+exports.default = function (app) {
+  app.get("/projects", function (req, res) {
+    res.send("all projects");
+  });
 
-var _dbConnection2 = _interopRequireDefault(_dbConnection);
+  app.post("/projects", function (req, res) {
+    var _req$body = req.body,
+        name = _req$body.name,
+        description = _req$body.description,
+        goals = _req$body.goals,
+        scopes = _req$body.scopes,
+        requirements = _req$body.requirements,
+        mentor = _req$body.mentor,
+        company = _req$body.company;
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+    res.send("create project with data: " + name + ", " + description + ", " + goals + ", " + scopes + ", " + requirements + ", " + mentor + ", " + company);
+  });
 
-var db = new _dbConnection2.default();
+  app.put("/projects", function (req, res) {
+    var _req$body2 = req.body,
```

Comment:
Same comment as for the route above, but even if `req.body` would be assigned to a variable, there's no need to add a numeric suffix as this is a different block/scope.